portfolio/src/main/webapp/script.js

```diff
@@ -16,7 +16,7 @@
  * Adds comments from /data
  */
 async function getComments() {
-  const commentsData = await fetch('/data');
+  const commentsData = await fetch("/data?max=5");
```

Comment:
I think either single quote, double quote is fine, but please keep style consistent across the repository (using single quote elsewhere).

Reply:
Sorry, that was a merging oversight

Reply:
Note: single quote recommended here https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes